packages/block-editor/src/components/keyboard-shortcuts/index.js

```diff
@@ -155,6 +155,15 @@ function KeyboardShortcutsRegister() {
 				character: 'g',
 			},
 		} );
+		registerShortcut( {
+			name: 'core/block-editor/zoom',
+			category: 'global',
+			description: __( 'Toggle zoom out.' ),
```

Comment:
```suggestion
			description: __( 'Enter or exit zoom out.' ),
```

There's a proposal to remove "Toggle" verb occurrences. See #66369.